roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Install latest st2chatops package
+  become: yes
+  package:
+    name: st2chatops
+    state: latest
+  when: st2chatops_version == "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Install pinned st2chatops package
+  become: yes
+  package:
+    name: st2chatops={{ st2chatops_version }}
+    state: present
+  when: st2chatops_version != "latest"
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Add st2_api_key in st2chatops.env
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '(?<=ST2_API_KEY=)(.*)$'
+    replace: '{{ st2chatops_ST2_API_KEY }}'
+  when: st2chatops_ST2_API_KEY != "CHANGE-ME-PLEASE"
+  register: st2chatops_api_key
+  tags: st2chatops
+
+- name: Comment Username, Password and Auth URL, if API_KEY provided
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '{{ item }}'
+    replace: '# \1'
+  with_items:
+    - '^(export ST2_AUTH_URL.*)'
+    - '^(export ST2_AUTH_USERNAME.*)'
+    - '^(export ST2_AUTH_PASSWORD.*)'
+  when: st2chatops_api_key.changed
+  tags: [st2chatops, skip_ansible_lint]
+
+- name: Uncomment Hubot "{{ st2chatops_HUBOT_ADAPTER|upper }}" adapter in "/opt/stackstorm/chatops/st2chatops.env"
+  become: yes
+  replace:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    regexp: '^# (export HUBOT_ADAPTER={{ st2chatops_HUBOT_ADAPTER }})$'
+    replace: '\1'
+  when: st2chatops_api_key.changed and st2chatops_HUBOT_ADAPTER in supported_hubot_adapter
+  register: uncomment_adapter
+  tags: [st2chatops, skip_ansible_lint]
+
+
+- name: Configure "{{ st2chatops_HUBOT_ADAPTER|upper }}" adapter setttings in "/opt/stackstorm/chatops/st2chatops.env"
+  become: yes
+  ini_file:
+    dest: /opt/stackstorm/chatops/st2chatops.env
+    section: null
+    option: "export {{ _conf_option.key }}"
+    value: "{{ _conf_option.value }}"
+    no_extra_spaces: yes
+  with_dict: "{{ st2chatops_config }}"
+  loop_control:
+    loop_var: _conf_option
+  when: uncomment_adapter.changed  # TODO: Add check for Individual keys in hash
```

Comment:
This will fail in cases when user can re-configure adapter settings.
Ex: change API key, but didn't change adapter.

Consider this Ansible role not as a way to install `st2chatops` role only once, but in a wide conext of management where different values could be changed/reconfigured with time.